noir/noir-repo/acvm-repo/bn254_blackbox_solver/src/fixed_base_scalar_mul.rs

```diff
@@ -1,3 +1,4 @@
+// TODO(https://github.com/AztecProtocol/aztec-packages/issues/6058): rename this file to something more generic
```

Comment:
ditto on issues location.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/6039/commits/f93a67f79a0cf01792953ca5fae469e303d8e36b